src/assets/stylesheets/layouts/layout-auth.scss

```diff
@@ -0,0 +1,42 @@
+.layout-auth {
+  &,
+  body,
+  .root-container {
+    height: 100%;
+  }
+
+  .root-container {
+    display: flex;
+  }
+
+  body {
+    background: {
+      image: url('/assets/images/background.png');
+      size: cover;
+
+      position: center;
+    }
+  }
+
+  .app-header-title {
+    padding-top: 2rem;
```

Comment:
Could you recheck this value? In the design, it's just `24px`, but you're setting `32px`
![Screenshot_13_02_2023__17_11](https://user-images.githubusercontent.com/7344405/218430245-b40ca2dd-801c-46b1-a4b8-aa291e2b7141.png)
```suggestion
    padding-top: rem(24px);
```

Check other spacing as well 🙏

Reply:
Fixed in https://github.com/suphanatjarukulgowit/react-nimble-survey/commit/f5c1469821f5cd51510a802528ec903b4dd6df28 and https://github.com/suphanatjarukulgowit/react-nimble-survey/pull/32/commits/e4f0bde54a85b5d0b7f0926ee32c3935774d2813